types/bun/globals.d.ts

```diff
@@ -170,7 +170,7 @@ declare var process: Process;
 interface BlobInterface {
   text(): Promise<string>;
   arrayBuffer(): Promise<ArrayBuffer>;
-  json(): Promise<JSON>;
+  json<TJSONReturnType>(): Promise<TJSONReturnType>;
```

Comment:
can we make it optional or otherwise, provide a default?

Reply:
@Jarred-Sumner For sure! I was going to have it extend `Record<string, unknown>` and return a default. Do you have any preferences over an empty object `{}` for a default?

Apologies, I banged it together in my lunch break and was going to get back to it a bit later.

Reply:
unknown is probably better since json can also be a array

Reply:
Thanks @Tricked-dev. I agree that's the best option.